.github/workflows/workflow.yml

```diff
@@ -247,3 +247,23 @@ jobs:
           push_image_and_stages: on:push
           username: ocamldune
           password: "${{ secrets.DOCKER_HUB_PASSWORD }}"
+
+  create-local-opam-switch:
+    name: Create local opam switch
+    strategy:
+      fail-fast: false
```

Comment:
why not `true`? AFAIK, this is to stop running jobs after new pushes?

Reply:
changed it to `true`